booking.html

```diff
@@ -28,120 +28,118 @@
+                <div class="session-payment">
+                    <p class="p2">Pay by</p>
+                    <div class="detail" id="photo-amount">
+                        <div class="by-photo-amount">
+                            <input id="photo-amount-radio" type="radio" name="detail" value="photo amount">
+                            <label class="payment-label" id="photo-amount-label" for="photo-amount-radio">By amount of
+                                photos</label>
+                        </div>
+                        <label for="photo-amount-dropdown"></label>
+                        <select id="photo-amount-dropdown">
+                            <option value="10 photos">10 photos 900 NOK</option>
+                            <option value="20 photos">20 photos 1.500 NOK</option>
+                            <option value="30 photos">30 photos 3.500 NOK</option>
                         </select>
-                    </label>
-
+                    </div>
+                    <div class="detail">
+                        <input id="half-day" type="radio" name="detail" value="half day">
+                        <label class="payment-label" id="half-day-label" for="half-day">Half day</label>
+                    </div>
+                    <div class="detail">
+                        <input id="full-day" type="radio" name="detail" value="full day">
+                        <label class="payment-label" id="full-day-label" for="full-day">Full day</label>
+                    </div>
+                    <div class="detail" id="photographers">
+                        <label id="photographer-dropdown-label" for="photographer-dropdown">Preferred
+                            photographer:</label>
+                        <div class="photographer-selection">
+                            <select id="photographer-dropdown">
+                                <option value="Panida Paethanom">Panida Paethanom</option>
+                                <option value="Helene Flått">Helene Flått</option>
+                                <option value="Kenneth Sheikh">Kenneth Sheikh</option>
+                                <option value="Berken Ates">Berken Ates</option>
+                                <option value="Patrick Røthe">Patrick Røthe</option>
+                                <option value="No preference">No preference</option>
+                            </select>
+                            <a id="photographer-link" href="#">See our photographers</a>
+                        </div>
+                    </div>
                 </div>
-                <!-- </form> -->
-            </section>
-
-            <div class="requests-form">
-                <p class="request-title">Other request & information</p>
-                <label for="message">Message:</label>
-                <textarea id="message" name="message" rows="3" required placeholder="Extra information"></textarea>
+            </div <div class="request">
```

Comment:
div is not closed properly.

Reply:
for some reason this does not show in my code program, but i wrote the line again with closing tags. to be sure